backend/src/main/scala/com/just/donate/models/Organisation.scala

```diff
@@ -240,7 +253,38 @@ case class Organisation(
       accounts = accounts.updated(fromAccount.name, updatedFrom).updated(toAccount.name, updatedTo)
     )
 
-    Right((updatedOrg, Seq.empty[EmailMessage]))
+    val queues = Seq(updatedFrom.unboundDonations) :++ updatedFrom.boundDonations.map(_._2).toSeq
```

Comment:
I feel like emails should be send somewhere else and under different conditions, or? Parts of a donation can be split and land on different accounts, this isnt a good indicator to send a mail.